src/mlpack/core/data/detect_file_type.cpp

```diff
@@ -74,7 +74,7 @@ arma::file_type GuessFileType(std::istream& f)
 
   // Handle empty files.
   if (nMax == 0)
-    return arma::file_type_unknown;
+    return file_type::FileTypeUnknown;
 
   const arma::uword nUse = std::min(nMax, arma::uword(4096));
```

Comment:
Change all instances of `arma::uword` to `size_t`, for consistency with the rest of the mlpack codebase